wordcloud/wordcloud.py

```diff
@@ -728,6 +730,130 @@ def __array__(self):
+            height = self.mask.shape[0]
+        else:
+            height, width = self.height, self.width
+
+        # Get max font size
+        if self.max_font_size is None:
+            max_font_size = max(w[1] for w in self.layout_)
+        else:
+            max_font_size = self.max_font_size
+
+        # Text buffer
+        result = []
+
+        # Prepare global style
+        style = {}
+        font = ImageFont.truetype(self.font_path, int(max_font_size * self.scale))
+        font_family, font_style = font.getname()
+        # TODO properly escape/quote this name
+        # TODO should add option to specify URL for font (i.e. WOFF file)
+        # TODO should maybe add option to embed font in SVG file
+        # TODO when embedding, we should try to embed only a subset
+        style['font-family'] = repr(font_family)
+        font_style = font_style.lower()
+        if 'bold' in font_style:
+            style['font-weight'] = 'bold'
+        if 'italic' in font_style:
+            style['font-style'] = 'italic'
+        elif 'oblique' in font_style:
+            style['font-style'] = 'oblique'
+        style = ';'.join(':'.join(pair) for pair in style.items())
+
+        # Add header
+        result.append(
+            '<svg'
+            ' xmlns="http://www.w3.org/2000/svg"'
+            ' width="{}"'
+            ' height="{}"'
+            ' style="{}"'
+            '>'
+            .format(width * self.scale, height * self.scale, style)
+        )
+
+        # Add background
+        if self.background_color is not None:
+            result.append(
+                '<rect'
+                ' width="100%"'
+                ' height="100%"'
+                ' style="fill:{}"'
+                '>'
+                '</rect>'
+                .format(self.background_color)
+            )
+        
+        # Embed image, useful for debug purpose
+        if embed_image:
+            image = self.to_image()
+            data = io.BytesIO()
+            image.save(data, format='JPEG')
+            data = base64.b64encode(data.getbuffer()).decode('ascii')
+            result.append(
+                '<image'
+                ' width="100%"'
+                ' height="100%"'
+                ' href="data:image/jpg;base64,{}"'
+                '/>'
+                .format(data)
+            )
+
+        # For each word in layout
+        for (word, count), font_size, (y, x), orientation, color in self.layout_:
+            x *= self.scale
+            y *= self.scale
+            
+            # Get text metrics
+            font = ImageFont.truetype(self.font_path, int(font_size * self.scale))
+            (size_x, size_y), (offset_x, offset_y) = font.font.getsize(word)
+            ascent, descent = font.getmetrics()
+
+            # Compute text bounding box
+            # TODO some browser do not render glyphs the same way (e.g. Segoe Script in Chrome is different, while in Internet Explorer it matches to_image)
```

Comment:
yes, it looks like there are questions of ligatures and joining forms... not sure we want to fix this, but should document it.